protocol/src/main/java/org/geysermc/mcprotocollib/network/tcp/DropPacketFilter.java

```diff
@@ -0,0 +1,23 @@
+package org.geysermc.mcprotocollib.network.tcp;
+
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelOutboundHandlerAdapter;
+import io.netty.channel.ChannelPromise;
+
+public class DropPacketFilter extends ChannelOutboundHandlerAdapter {
```

Comment:
I understand why it's named filter, but later in the pipeline it's referred to as "packet_filter" which imo doesn't quite convey the functionality of the pipeline "flushing" (why'd you rename it to dropping anyways? Just curious)

Maybe `FlushHandler` or something?

Reply:
Yeah idk about the inconsistency, I have renamed and replaced this thing like 5 times until I got this working impl. I'll rename it to FlushHandler.